virttest/utils_misc.py

```diff
@@ -2296,38 +2296,35 @@ def get_qemu_dst_binary(params):
     return qemu_dst_binary
 
 
+def get_binary(binary_name, params):
+    """
+    Get the path to the binary currently in use.
+    """
+    binary_path = get_path(_get_backend_dir(params),
+                           params.get("%s_binary" % binary_name, binary_name))
+    if not os.path.isfile(binary_path):
+        logging.debug('Could not find params %s in %s, searching the '
+                      'host PATH for one to use',
+                      binary_name,
+                      binary_path)
+        binary_path = utils_path.find_command(binary_name)
+        logging.debug('Found %s', binary_path)
+
+    return binary_path
+
+
 def get_qemu_img_binary(params):
     """
     Get the path to the qemu-img binary currently in use.
     """
-    qemu_img_binary_path = get_path(_get_backend_dir(params),
-                                    params.get("qemu_img_binary", "qemu-img"))
-    if not os.path.isfile(qemu_img_binary_path):
-        logging.debug('Could not find params qemu-img in %s, searching the '
-                      'host PATH for one to use', qemu_img_binary_path)
-        qemu_img_binary = utils_path.find_command('qemu-img')
-        logging.debug('Found %s', qemu_img_binary)
-    else:
-        qemu_img_binary = qemu_img_binary_path
-
-    return qemu_img_binary
+    return get_binary('qemu-img', params)
```

Comment:
Gievn `qemu-img`, `params.get("%s_binary" % binary_name, binary_name)` would be `qemu-img_binary`.

Reply:
Good Catch. Updated, thanks